telegram-bot/src/main/java/ru/otus/flightsearch/component/FlightSearcherBot.java

```diff
@@ -41,11 +45,17 @@ public class FlightSearcherBot extends TelegramLongPollingBot {
     @Override
     public void onUpdateReceived(Update update) {
 
-
         if (update.hasMessage() && update.getMessage().hasText()) {
-            String inputMessage = update.getMessage().getText();
+            Message message = update.getMessage();
+            String inputMessage = message.getText();
 
             if (inputMessage.startsWith(SHOW_TICKETS)) {
+                botBuyerService.postBuyerInfo(
```

Comment:
do we need to update buyerInfo every time tickets are requested?

Reply:
Not really, and that's precisely i wanted to ask you.

it should be done one time for one user session. Need to think about implementation.

Reply:
t.b.d. draft